wled00/wled.h

```diff
@@ -565,7 +564,7 @@ WLED_GLOBAL JsonDocument* fileDoc;
 WLED_GLOBAL bool doCloseFile _INIT(false);
 
 // presets
-WLED_GLOBAL int16_t currentPreset _INIT(-1);
+WLED_GLOBAL int8_t currentPreset _INIT(-1);
```

Comment:
Presets can go up to 250, `int8_t` only 127.

Reply:
I overlooked that. Thanks.

Reply:
Hmmm. Thinking of it, there is no preset #0.
Preset 0 usualy means undefined/not used.

Reply:
There was _some reason_ I defined no preset as -1 instead of 0, but I can neither remember what it was nor see it in the code at quick glance. If there are no negative consequences, changing "no preset" to 0 and setting type as `uint8_t` would make sense.

Reply:
Remembered! PL=0 used to be "return to boot defaults", but that has since been removed. We can go ahead and change this.

Reply:
Done! JSON API will still return -1 to avoid breaking api change.